test/sandbox-test.js

```diff
@@ -113,6 +113,13 @@ describe("sinon.sandbox", function () {
                     assert(sinon.useFakeXMLHttpRequest.called);
                 });
 
+                it("doesn't secretly use useFakeServer", function () {
+                    this.sandbox.stub(sinon.fakeServer, "create").returns({ restore: function () {} });
```

Comment:
This seems like a bad test. Did this test fail prior to this change? By default, sinon uses `sinon.fakeServerWithClock` to construct fake servers.